helm_deploy/hmpps-prisoner-to-nomis-update/values.yaml

```diff
@@ -169,3 +169,6 @@ generic-prometheus-alerts:
   targetApplication: hmpps-prisoner-to-nomis-update
   alertSeverity: syscon
   cronjobTargetOverride: "hmpps-prisoner-to-nomis-update-queue-.*"
+
+cron:
+  allocationReconReportSchedule: "5 23 * * *"
```

Comment:
Agreed with the Activities team that this is the best time to run in prod - after their nightly job which handles ended / pending allocations.